Source/Core/Core/HW/EXI_DeviceIPL.cpp

```diff
@@ -180,20 +180,15 @@ std::string CEXIIPL::FindIPLDump(const std::string& path_prefix)
   return ipl_dump_path;
 }
 
-void CEXIIPL::LoadFontFile(const std::string& filename, u32 offset)
+void CEXIIPL::LoadFontFile(const File::Path& path, u32 offset)
 {
   // Official IPL fonts are copyrighted. Dolphin ships with a set of free font alternatives but
   // unfortunately the bundled fonts have different padding, causing issues with misplaced text
   // in some titles. This function check if the user has IPL dumps available and load the fonts
   // from those dumps instead of loading the bundled fonts
 
-  // Check for IPL dumps in User folder
   std::string ipl_rom_path = FindIPLDump(File::GetUserPath(D_GCUSER_IDX));
 
-  // If not found, check again in Sys folder
-  if (ipl_rom_path.empty())
-    ipl_rom_path = FindIPLDump(File::GetSysDirectory() + GC_SYS_DIR);
-
```

Comment:
I'm intentionally dropping this "feature" of reading fonts from IPL dumps in the Sys directory. (IPL dumps in the user directory still work fine.) There is no reason that we would provide a Sys directory with an IPL dump containing a font that's better than the one in the font file, and users aren't supposed to put things in the Sys directory. (This isn't dropped just for the sake of it - it's done because re-implementing it using the new way of accessing the Sys directory would be a bit annoying.)